server/room-game.ts

```diff
@@ -62,13 +62,15 @@ export class RoomGamePlayer<GameClass extends RoomGame = SimpleRoomGame> {
 	 * `this.getUser().games`.
 	 */
 	readonly id: ID;
+	completed: boolean;
```

Comment:
```suggestion
	completed?: boolean;
```
I would probably make this optional.